File: GTTS/INL.VettingService.Models/SaveVettingBatches_Result.cs
using INL.Services.Models;
using System.Collections.Generic;

namespace INL.VettingService.Models
{
    public class SaveVettingBatches_Result : BaseResult, ISaveVettingBatches_Result
    {
        public List<IVettingBatch_Item> Batches { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/save-vetting-batches_result.ts


import { IVettingBatch_Item } from './ivetting-batch_item';
import { ISaveVettingBatches_Result } from './isave-vetting-batches_result';

export class SaveVettingBatches_Result implements ISaveVettingBatches_Result {
  
	public Batches?: IVettingBatch_Item[];
  
	public ErrorMessages: string[] = null;
}


